### DISCUSSION TITLE
⏳ 🔗 ACF Repeater, Group Need to be Implemented (Duplicate)

### DISCUSSION THREAD

**Discussion by felmerald** [elementor/elementor#22833] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor needs to workout the implementation of ACF repeater, grouping with conditional logic (Hidden  / Show if the elements is empty or not) this would be very powerful functionality.

### Describe the solution you'd like

Elementor needs to workout the implementation of ACF repeater, grouping with conditional logic (Hidden  / Show if the elements is empty or not) this would be very powerful functionality.

**Comment by lensbreak** [comment]
+1 very much needed especially for any ecommerce website. example: you can create a video gallery (together with the new loop builder)

**Reply by felmerald** [reply]
Yes, @lensbreak  Elementor needs to implement an ACF repeater, and grouping this implementation would be much more powerful. Is anyone have ideas if the Elementor is planning on this?

**Reply by lensbreak** [reply]
usually they wait 1-2 years and first focus on changing colors and names of their UI panels..

**Comment by sarends08** [comment]
+1 - This would be a huge improvement for our team. I run into this issue quite frequently. It makes our client sites look sloppy in the WordPress Editor and takes up additional implementation time.

**Comment by brahimvandenbrande** [comment]
+1! I completely agree. It would be beneficial to integrate ACF Flexible content natively without relying on external plugins. This would streamline the process and enhance the user experience. Thanks for suggesting this improvement!

**Comment by ayoubkhan558** [comment]
There is already a discussion of something like this.
- Enable Repeater Fields in Loop Grid and Loop Carousel Widgets
#22170
